examples/solvers/chambolle_pock_denoising.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Denoising using Chambolle-Pock solver."""
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+
+# External
+import numpy as np
+import scipy
+import scipy.ndimage
+import matplotlib.pyplot as plt
+
+# Internal
+import odl
+from odl.solvers import chambolle_pock_solver, f_cc_prox_l2_tv, g_prox_none
+
+
+# Read the image
+im = np.rot90(scipy.misc.lena()[::2, ::2], 3)
```

Comment:
Why so overly short? I think we should be extra good with variable names and other stylistic things in examples. Use `image`.